bitnami/redis-cluster/values.yaml

```
@@ -345,9 +385,12 @@ redis:
   ## ref: https://redis.io/topics/cluster-tutorial#creating-and-using-a-redis-cluster
   ##
   useAOFPersistence: "yes"
-  ## @param redis.port Redis&trade; port
+  ## @param redis.ports.redis Redis&trade; port
+  ## @param redis.ports.bus The busPort should be obtained adding 10000 to the redisPort. By default: 10000 + 6379 = 16379
   ##
-  port: 6379
+  ports:
```

This property is standardize to `containerPorts`:
```suggestion
  containerPorts:
```

---

Please note this change implies adapting your whole PR